core/src/main/java/com/atlassian/migration/datacenter/core/aws/CfnApi.java

```
@@ -132,6 +191,7 @@ public StackStatus getStatus(String stackName) {
             Stack stack = response.stacks().get(0);
             return Optional.ofNullable(stack);
         } catch (CompletionException | CancellationException e) {
+            logger.error("Error getting stack {}", stackName, e);
             return Optional.empty();
```

Not part of this PR, but shouldn't this return an error with a message so that the UI can present the information?

---

We have a ticket open to address this method. I think because we're using exceptions in most places for error stages, we should just throw an exception. https://aws-partner.atlassian.net/browse/CHET-219

I didn't want to tackle it here because it's kind of an extra thing. I agree though it's not very good traceability into what's going on.